src/main/resources/templates/db_scripts/schema.json

```diff
@@ -215,6 +215,14 @@
       "fromModuleVersion": "mod-organizations-storage-4.7.0",
       "withOptimisticLocking": "failOnConflict",
       "withMetadata": true,
+      "fullTextIndex": [
+        {
+          "fieldName": "bankAccountNumber",
+          "tOps": "ADD",
```

Comment:
maybe lets remain only fieldName and tpOps

Full text search index was chosen, because
= queries usually do full text search. == queries with "**" do like search which gin index required

Reply:
done